test/clj/rems/test/api/applications.clj

```diff
@@ -869,3 +869,29 @@
                   "event/decided"
                   "event/approved"]
                  (map :event (get-in data [:application :dynamic-events])))))))))
+
+(deftest dynamic-application-create-test
```

Comment:
Btw. should we change to the `test-` naming convention sometime?

Reply:
I think @luontola changed all of these to `-test` a while ago. I prefer `test-` but followed the convention in this file.

Reply:
So maybe a separate PR sometime later.